.github/ISSUE_TEMPLATE/iceberg_bug_report.yml

```diff
@@ -38,14 +39,6 @@ body:
         - "1.2.0"
         - "1.1.0"
         - "1.0.0"
-        - "0.14.1"
```

Comment:
I removed some older versions from issue template as we ask users to reproduce in the latest version anyway if the version is too old!